src/app/shared/services/chat/chat-msg.model.ts

```diff
@@ -0,0 +1,26 @@
+export interface IRapidProMessage {
+    message: string;
+    message_id: string;
+    title: string;
+    type: string;
+    wasTapped: boolean; // informs whether message received in app foreground or background
+    quick_replies?: string; // string with JSON array. e.g "["English","Malay"]"
```

Comment:
nice to know that is the format. If you find this happening in lots of places or want to make it more explicit you could always define an alternate string type type, e.g. `type stringifiedJson = string` and then just say `quick_replies?: stringifiedJson`
But equally the comment achieves that